spec/lib/tracked_revisions_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require_dependency "#{Rails.root}/lib/analytics/course_statistics.rb"
+
+describe 'TrackedRevisions' do
```

Comment:
This could be named more clearly, perhaps `Course#tracked_revisions` since that's the exact scope it's mainly testing.

Probably this can go in `spec/models` in that case.